sdk/core/System.ClientModel/src/Pipeline/PipelineTransport.cs

```diff
@@ -97,9 +145,13 @@ private async ValueTask ProcessSyncOrAsync(PipelineMessage message, bool async)
                 ProcessCore(message);
             }
         }
-        catch (OperationCanceledException ex)
+        catch (Exception ex)
```

Comment:
This looks like we added the conditional but no default logic and the end result is the same as the previous form.   Was that intentional?

Reply:
haha yeah let me fix this. I was writing logs here but it was missing some exception cases so moved it to L111